ci/actions-templates/centos-fmt-clippy-template.yaml

```diff
@@ -49,14 +49,7 @@ jobs: # skip-all
           restore-keys: ${{ github.base_ref }}-${{ runner.os }}-cargo-clippy-target-dir-${{ steps.cargo-target-cache.outputs.rust_hash }}-${{ hashFiles('**/Cargo.lock') }}
       - name: Install Rustup using ./rustup-init.sh
         run: |
-          sh ./rustup-init.sh --default-toolchain=none --profile=minimal -y
-      - name: Ensure Beta is up to date
-        run: |
-          if rustc +beta -vV >/dev/null 2>/dev/null; then
-            rustup toolchain uninstall beta
-          fi
-          rustup toolchain install --profile=minimal beta
-          rustup default beta
+          sh ./rustup-init.sh --default-toolchain=stable --profile=minimal -y
```

Comment:
Kinda unrelated, but I assume this would be more efficient if we just install the full toolchain directly rather than adding rustfmt/clippy at a later stage?

Reply:
@djc Thanks!

Reply:
FWIW, I would have made this a separate commit.